lib/node/rpc.js

```diff
@@ -2346,10 +2347,95 @@ class RPC extends RPCBase {
     return result;
   }
 
+  async deriveAddresses(args, help) {
+    if (help || args.length > 2 || args.length === 0)
+      throw new RPCError(errs.MISC_ERROR, 'deriveaddresses "descriptor"');
```

Comment:
Is range an optional param? I think it should be,

```suggestion
      throw new RPCError(errs.MISC_ERROR, 'deriveaddresses "descriptor"' (range));
```